setup.py

```diff
@@ -55,6 +55,9 @@ def run_tests(self):
     "urllib3 <2; python_version <'3.10'",
     # https://github.com/kevin1024/vcrpy/pull/775#issuecomment-1847849962
     "urllib3 <2; platform_python_implementation =='PyPy'",
+    # Workaround for Poetry with Python >= 3.10, problem description at:
```

Comment:
@pjonsson maybe "Python" could better read "CPython" here.

Reply:
You're right, fixed.

Reply:
@pjonsson thanks!